### ISSUE TITLE
metric_NMI

### ISSUE THREAD

**Issue by niklasmueboe** [SpatialHackathon/SpaceHack2023#2]
Normalized mutual information

- https://dl.acm.org/doi/10.5555/1756006.1953024
- https://scikit-learn.org/stable/modules/generated/sklearn.metrics.normalized_mutual_info_score.html
